test/Microsoft.Extensions.CommandLineUtils.Tests/CommandLineApplicationTests.cs

```diff
@@ -400,7 +400,7 @@ public void HandlesAsyncExecutes()
 
             app.OnExecute(async () =>
             {
-                await Task.Delay(5);
+                await Task.Delay(5).ConfigureAwait(false);
```

Comment:
Lmao, ya nuke these tests. Can't beleive we're validating that it can be executed asynchronously.